projects/bp-gallery/src/components/views/search/AdvancedSearch.tsx

```diff
@@ -0,0 +1,277 @@
+  );
+
+  const buildAttributeFilter = useCallback(
+    (attributeFilterProps: AttributeFilterProps) => {
+      const filter = `${attributeFilterProps.filterProps.reduce(
+        (accumulator, currentValue) =>
+          `${accumulator}${buildSingleFilter(
+            attributeFilterProps.attribute,
+            currentValue.filterOperator,
+            currentValue.values[0],
+            currentValue.values[1],
+            currentValue.combinationOperator
+          )}`,
+        ''
+      )}`;
+      return filter !== '' ? `(${filter})` : '';
+    },
+    [buildSingleFilter]
+  );
+
+  const filter: string = advancedSearchProps
+    .map(attributeFilterProps => buildAttributeFilter(attributeFilterProps))
+    .reduce((accumulator, currentvalue) => {
+      if (accumulator !== '' && currentvalue === '') {
+        return accumulator;
+      } else if (accumulator === '' && currentvalue !== '') {
+        return currentvalue;
+      } else if (accumulator !== '' && currentvalue !== '') {
+        return `${accumulator} AND ${currentvalue}`;
+      } else {
+        return '';
+      }
+    }, '');
+
+  // no in use as long as the comment index doesn't work properly
+  // const searchIndices = ['picture', 'comment'];
+
+  return (
+    <div className='flex flex-col m-auto w-fit'>
+      <div className='breadcrumb m-1'>
+        <SearchBreadcrumbs searchParams={searchParams} />
+      </div>
+      <div className='flex flex-row'>
+        <div className='search-bar-container shadow'>
+          <SearchBar searchParams={searchParams} isAllSearchActive={isAllSearchActive} />
+          <Accordion
+            key={0}
+            disableGutters={true}
+            square={true}
+            sx={{
+              backgroundColor: '#e9e9e9',
+              boxShadow: '0px -1px 0px rgba(0, 0, 0, 0.3)',
+              width: 'fit-content',
+            }}
+          >
+            <AccordionSummary expandIcon={<ExpandMore />}>
+              <div className='flex flex-row flex-nowrap items-center'>
+                <Typography fontWeight='bold'>{t('search.advanced-search-title')}</Typography>
+                <HelpTooltip
+                  title={t('search.advanced-question')}
+                  content={t('search.advanced-help')}
+                />
+              </div>
+            </AccordionSummary>
+            <div className='advanced-search w-fit p-4 flex flex-col'>
+              {/* <div className='flex flex-row flex-nowrap justify-start items-center'>
+                <Typography fontWeight={'bold'}>{t(`search.setIndex`)}</Typography>
+                <Select value={searchIndex} onChange={event => setSearchIndex(event.target.value)}>
+                  {searchIndices.map(searchIndex => (
+                    <MenuItem key={searchIndex} value={searchIndex}>
+                      {t(`search.${searchIndex}`)}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </div> */}
```

Comment:
still needed?

Reply:
yes this may come back in use, if I manage to make the comment index work